src/main/java/com/cst438/domain/Assignment.java

```diff
@@ -11,9 +11,40 @@ public class Assignment {
     @GeneratedValue(strategy=GenerationType.IDENTITY)
     @Column(name="assignment_id")
     private int assignmentId;
- 
-    // TODO  complete this class
-    // add additional attributes for title, dueDate
```

Comment:
All the necessary changes were made and erroneous reversion has been resolved. Great work.